boot.php

```diff
@@ -1276,27 +1247,26 @@ function is_friendica_app() {
 	 * @return bool Is it a known backend?
 	 */
 	function is_backend() {
-		static $backend = array(
-			"_well_known",
-			"api",
-			"dfrn_notify",
-			"fetch",
-			"hcard",
-			"hostxrd",
-			"nodeinfo",
-			"noscrape",
-			"p",
-			"poco",
-			"post",
-			"proxy",
-			"pubsub",
-			"pubsubhubbub",
-			"receive",
-			"rsd_xml",
-			"salmon",
-			"statistics_json",
-			"xrd",
-		);
+		$backend = array();
+		$backend[] = "_well_known";
+		$backend[] = "api";
+		$backend[] = "dfrn_notify";
+		$backend[] = "fetch";
+		$backend[] = "hcard";
+		$backend[] = "hostxrd";
+		$backend[] = "nodeinfo";
+		$backend[] = "noscrape";
+		$backend[] = "p";
+		$backend[] = "poco";
+		$backend[] = "post";
+		$backend[] = "proxy";
+		$backend[] = "pubsub";
+		$backend[] = "pubsubhubbub";
+		$backend[] = "receive";
+		$backend[] = "rsd_xml";
+		$backend[] = "salmon";
+		$backend[] = "statistics_json";
+		$backend[] = "xrd";
```

Comment:
Can you explain why you changed this variable assignation? It is neither a visual or a performance improvement (4 times slower).